tools/Tools.Common/Loaders/CmdletLoader.cs

```diff
@@ -20,47 +20,32 @@
 using Tools.Common.Models;
 using Tools.Common.Extensions;
 using System.IO;
+using System.Runtime.Loader;
 
 namespace Tools.Common.Loaders
 {
-// TODO: Remove IfDef
-#if NETSTANDARD
     public class CmdletLoader
-#else
-    public class CmdletLoader : MarshalByRefObject
-#endif
     {
         public static ModuleMetadata ModuleMetadata;
 
-        public ModuleMetadata GetModuleMetadata(string assemblyPath, List<string> commonOutputFolders)
-        {
-            AppDomain.CurrentDomain.AssemblyResolve += (sender, args) =>
-            {
-                foreach (var commonOutputFolder in commonOutputFolders)
-                {
-                    var assemblyName = args.Name.Substring(0, args.Name.IndexOf(","));
-                    var dll = Directory.GetFiles(commonOutputFolder, "*.dll").FirstOrDefault(f => Path.GetFileNameWithoutExtension(f) == assemblyName);
-                    if (dll == null)
-                    {
-                        continue;
-                    }
-
-                    return Assembly.LoadFrom(dll);
```

Comment:
`Assembly.LoadFrom()` will load the dll into the default ALC, so the dlls may conflict.